receiver/dummyreceiver/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+type: dummy
+
+status:
+  class: receiver
+  stability:
+    alpha: [metrics]
+  distributions: [kyma]
+  codeowners:
+    active: [kyma-project/observability]
+
+resource_attributes:
+  k8s.cluster.name:
+    description: "static value: test-cluster"
+    type: string
+
+attributes:
+  host:
+    description: the current hostname
+    type: string
+
+metrics:
+  dummy:
+    description: a dummy gauge
```

Comment:
we should add  `metric.SetDescription("This is a dummy metric")` [here](https://github.com/kyma-project/opentelemetry-collector-components/blob/42ae0739db0739728900ac7589c30f8986da2f62/receiver/dummyreceiver/metrics_receiver.go#L75)

Reply:
done